sdks/wasm/Makefile

```diff
@@ -126,7 +126,7 @@ MONO_LIBS = $(TOP)/sdks/out/wasm-runtime-release/lib/{libmono-ee-interp.a,libmon
 MONO_THREADS_LIBS = $(TOP)/sdks/out/wasm-runtime-threads-release/lib/{libmono-ee-interp.a,libmono-native.a,libmono-icall-table.a,libmonosgen-2.0.a,libmono-ilgen.a}
 MONO_DYNAMIC_LIBS = $(TOP)/sdks/out/wasm-runtime-dynamic-release/lib/{libmono-ee-interp.a,libmono-native.a,libmono-icall-table.a,libmonosgen-2.0.a,libmono-ilgen.a}
 
-EMCC_FLAGS=-s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s ALIASING_FUNCTION_POINTERS=0 -s NO_EXIT_RUNTIME=1 -s "EXTRA_EXPORTED_RUNTIME_METHODS=['ccall', 'FS_createPath', 'FS_createDataFile', 'cwrap', 'setValue', 'getValue', 'UTF8ToString', 'addFunction']" -s USE_ZLIB=1 -s "EXPORTED_FUNCTIONS=['_putchar']" --source-map-base http://example.com  -s WASM_OBJECT_FILES=0
+EMCC_FLAGS=-s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s ALIASING_FUNCTION_POINTERS=0 -s NO_EXIT_RUNTIME=1 -s "EXTRA_EXPORTED_RUNTIME_METHODS=['ccall', 'FS_createPath', 'FS_createDataFile', 'cwrap', 'setValue', 'getValue', 'UTF8ToString', 'addFunction']" -s USE_ZLIB=1 -s "EXPORTED_FUNCTIONS=['_putchar']" --source-map-base http://example.com  -s WASM_OBJECT_FILES=0 -s FORCE_FILESYSTEM=1
```

Comment:
@kjpou1 shouldn't this then be reverted? I added this here to force every mono.wasm ever built to include fs support, but we now have --enable-fs at packager.exe level. I'd like to know your feelings here :-)

Another important reason is we consume mono.wasm through your NuGets from Jenkins (nowadays), and want to preload stuff in a target or similar by calling file_packager.py, so we need such mono.wasm to have fs support.

Reply:
No, the runtime built from the makefile will be packaged through the NuGets and the SDK as well so it needs to be set here so that separate packaged files can be loaded.

The packager.exe is a separate process right now that builds its own runtime modules to be used.  If anything changes in the future we will have that option there to control export the filesystem methods.  When one uses Emscripten with the --preload-file option it will automatically add the FORCE_FILESYSTEM filesystem for you.

Reply:
Cristal clear, thanks man :-)